pyglider/seaexplorer.py

```diff
@@ -141,6 +141,26 @@ def raw_to_rawnc(indir, outdir, deploymentyaml, incremental=True, min_samples_in
     _log.info('All raw files converted to nc')
     return True
 
+
+def drop_rogue_1970(ds):
+    """
+    If dates greater than 1971, 1, 1 are observed, drop any dates before 1971-01-01, from the datset and return it.
+    This function removes 1970 timestamps caused by a SeaExplorer rebooting during a mission.
+    If all dates are < 1971-01-01, no action is taken
+
+    Parameters:
+        ds: xarray.DataSet
+            dataset to check for pre-1971 dates
+    Returns:
+        ds: xarray.DataSet
+    """
+    dt_1971 = pd.to_datetime(datetime.datetime(1971, 1, 1))
```

Comment:
Does np.datetime64("1971-01-01") work here?

Reply:
It does. I've switched to using numpy